src/Nethermind/Chains/gnosis.json

```diff
@@ -41,11 +41,14 @@
     }
   },
   "params": {
+    "blobGasPriceUpdateFraction": "0x10fafa",
```

Comment:
should we put into gnosis when we are not doing hardfork yet?

Reply:
What's your concern exactly? How can this be a problem?